### PR TITLE
chore: adds default acceptance criteria to feature template

### PR DESCRIPTION
**Pull request by mikaelbr**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new_feature.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
 
+## Acceptance Criteria
+
+_List of relevant acceptance criteria as a part of a QA flow_
 
+- [] Renders colors as expected in dark and light mode.
+- [] Is translatable to English and Norwegian.
+- [] Does not drain battery
+- [] Works in up to 200% font size
+- [] Is navigatable
```
Comment (rikkekantega) [inline .github/ISSUE_TEMPLATE/new_feature.md:33]: + Operable and perceivable using screen reader?

**Review by rikkekantega — APPROVED** [review]
Superb 🤩🙌

File: .github/ISSUE_TEMPLATE/new_feature.md (hunk 1 above)
Comment (mikaelbr) [reply .github/ISSUE_TEMPLATE/new_feature.md:33]: 🤦 I just found these changes as uncommited changes in my terminal and seems as I just stopped midway.... Thanks 😅 It was supposed to be "is navigatable using screen reader", but I like your wording better.

Comment (mikaelbr) [inline .github/ISSUE_TEMPLATE/new_feature.md:33]:
```suggestion
- [] Is operable and perceivable using screen reader
```

**Pull request merged**